### PR TITLE
Move form links and fix breadcrumbs for pensions and burials

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3323

This puts the pensions and burials forms in the right spot in the content hierarchy and fixes breadcrumbs in each.

Urls are now:

/pension/application/527EZ
/burials-and-memorials/application/530

screen shot 2017-06-21 at 5 46 26 pm
screen shot 2017-06-21 at 5 47 18 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@rroueche What should the breadcrumbs look like for these forms? I get a broken link error because /pension/application is not a page if I use the automatically generated breadcrumbs. Should that be a real page?

**Comment by rroueche** [comment]
Hmmm, now that is a good question.

Do you know how the breadcrumbs work for other tools? I actually don't think the other tools have breadcrumbs if memory serves...but I haven't checked yet. We should do a URL standardization activity this summer, but I'd say since we're trying to launch all these things 6/30, for their URLs we should go w/status quo for now.

**Comment by jbalboni** [comment]
@rroueche It looks like we manually create breadcrumbs for the other forms, and instead of "Application" it's "Apply for whatever", and that goes to the apply.md page.

Would it be better to put these forms in an apply/ folder (instead of application)? Or keep the url structure as it is now and just manually set the breadcrumbs?

**Comment by jbalboni** [comment]
I updated the pages to use manual breadcrumbs without the middle Application page. We can always change that if we need to.

**Comment by rroueche** [comment]
@jbalboni sorry was getting my PIV renewed (sigh). 

@peggygannon which static content pages will be linking off to these tools? For pension I believe it's: https://www.vets.gov/pension/apply/, but I'm not sure which burial & memorial page(s) will link to that application. Would be good for the breadcrumbs to reflect that path, so perhaps the URL and crumbs for Pension (for example) are:

/pension/apply/[form number]
Pension Benefits >  Application Process > [Name of Application Page]

**Review by cehsu — APPROVED** [review]
LGTM.

File: content/pages/burials-and-memorials/application/530.md [hunk 1]
```diff
@@ -1,17 +1,16 @@
 ---
-title: Apply for burial benefits
+title: Apply online
```
Comment (cvalarida) [inline content/pages/burials-and-memorials/application/530.md]: Is this the window title?

File: content/pages/pension/application/527EZ.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
 ---
-title: Pensions
+title: Apply for pension benefits
```
Comment (cvalarida) [inline content/pages/pension/application/527EZ.md]: For burials, the title is just "Apply online." Should that be consistent here?

File: script/build.js [hunk 3]
```diff
@@ -126,8 +126,8 @@ const ignore = require('metalsmith-ignore');
 const ignoreList = [];
 if (options.buildtype === 'production') {
   ignoreList.push('education/gi-bill/post-9-11/status.md');
-  ignoreList.push('pensions/application.md');
-  ignoreList.push('burials-and-memorials/application.md');
+  ignoreList.push('pension/application/527.md');
+  ignoreList.push('burials-and-memorials/application/530.md');
```
Comment (cvalarida) [inline script/build.js]: Why is `pension` singular but `burials and memorials` plural?

Not sure who to ask, so I'll flag @J-Quag (who can then flag whomever is the person to ask).

File: content/pages/pension/application/527EZ.md (hunk 2 above)
Comment (jbalboni) [inline content/pages/pension/application/527EZ.md]: Yeah, not sure why I did that.

File: content/pages/burials-and-memorials/application/530.md (hunk 1 above)
Comment (jbalboni) [inline content/pages/burials-and-memorials/application/530.md]: Yeah, also shows up in breadcrumbs potentially

**Comment by bethpotts** [comment]
@jbalboni I think all you need (which pages need links and related content changes) can be found in two docs Peggy put together last week in these issues. The relevant pages are eligibility and application process (apply) for Pension, and eligibility and allowances (/burials-and-memorials/survivor-and-dependent-benefits/burial-costs/) for Burials:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3320
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3319

**Review by cvalarida — APPROVED** [review]

**Pull request merged**